### PR TITLE
Update UI for navigating to a  bookmark.

### PR DESCRIPTION
**Pull request by hedgerwang**
Per feedback from users, make the UI that navigates to  a bookmark less confusing.

| Before| After (For Link) | After (For Bookmark) |
|---|---|---|
| image | image| image
 |

### CODE REVIEW

**Comment by velveret** [comment]
Has Design signed off on this? For regular links, users may want to be able to see the link contents without having to specifically hover the button

**Comment by hedgerwang** [comment]
re @velveret : we don't have a design for this.
I'll loop in @armanjaffer and see if he has any suggestions

**Comment by armanjaffer** [comment]
We had talked about removing the anchor tag label. https://app.asana.com/0/638227005585927/1122470366044592
Not super sure about the direction for link editing here. Mind if we do it in a separate diff @hedgerwang . @velveret I think I can sign off on the bookmark change (Sejal is out)

**Comment by hedgerwang** [comment]
re @velveret : For regular link, the UI look like this.

image

File: src/ui/LinkTooltip.js [hunk 1]
```diff
@@ -9,6 +9,10 @@ import CustomButton from './CustomButton';
 
 import './czi-link-tooltip.css';
 
+function isBookmarHref(href: string): boolean {
```
Comment (velveret) [inline src/ui/LinkTooltip.js]: Bookmar - Bookmark

File: src/ui/LinkTooltip.js [hunk 2]
```diff
@@ -26,19 +30,24 @@ class LinkTooltip extends React.PureComponent<any, any, any> {
 
   render(): ?React.Element<any> {
     const {href, editorView, onEdit, onRemove} = this.props;
+    const useBookMark = isBookmarHref(href);
+    const editButton = useBookMark ? null : (
```
Comment (velveret) [inline src/ui/LinkTooltip.js]: this could be `!useBookMark && ...` instead of the ternary

**Pull request merged**